src/views/Benchmark/index.jsx

```diff
@@ -10,23 +12,79 @@ class Benchmark extends Component {
 
   state = {
     platform: 'win10',
-    benchmark: 'ARES6',
+    benchmark: 'raptor-motionmark-animometer-firefox',
   }
 
-  onChange(event) {
+  async componentDidMount() {
+    const { platform, benchmark } = this.state;
+    this.fetchData(
+      CONFIG[platform].options.frameworkId,
+      CONFIG[platform].perfherderKey,
+      benchmark,
+      CONFIG[platform].options.buildType,
+    );
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    const { platform, benchmark } = this.state;
+    if (benchmark !== prevState.benchmark || platform !== prevState.platform) {
+      this.fetchData(
+        CONFIG[platform].options.frameworkId,
+        CONFIG[platform].perfherderKey,
+        benchmark,
+        CONFIG[platform].options.buildType,
+      );
+    }
+  }
+
+  async onChange(event) {
+    // Clear the plotted graphs
+    this.setState({ data: null });
     if (event.target.name === 'platform') {
       this.setState({
-        benchmark: CONFIG[event.target.value].benchmarks[0],
+        benchmark: Object.keys(CONFIG[event.target.value].benchmarks)[0],
       });
     }
     this.setState({ [event.target.name]: event.target.value });
   }
 
+  async fetchData(frameworkId, perfherderKey, benchmark, option) {
+    const { perfherderUrl, data } = await subbenchmarksData(
+      frameworkId,
+      perfherderKey,
+      benchmark,
+      option,
+    );
+    this.setState({ perfherderUrl, data });
+  }
+
   render() {
+    const { data } = this.state;
+
+    const sortAlphabetically = (a, b) => {
+      if (a.meta.test < b.meta.test) return -1;
```

Comment:
It is recommended to add braces because if you (or someone else) ever expands the statement it will be required.

```js
if (a.meta.test < b.meta.test) {
  return -1;
}
```